syntax="proto3";
package snail.stream;

message CommonReq {
    string reqid = 1;
    uint32 cluster_id = 2;
    map<string, string> headers = 3;
}

message CommonResp {
    string reqid = 1;
    uint32 cluster_id = 2;
    int32 code = 3;
    string reason = 4;
    map<string, string> headers = 5;
}
